skins/laika/src/components/pages/membership_form/Address.vue

```diff
@@ -152,19 +152,3 @@ export default Vue.extend( {
 	},
 } );
 </script>
-<style lang="scss" scoped>
-    @import "../../../scss/custom";
-
-    button.is-main {
-        height: 54px;
-        font-size: 1em;
-        font-weight: bold;
-        width: 250px;
```

Comment:
the only difference is that in buefy the width is 270px, I guess it doesn't matter that much.

Reply:
The CSS selector for main buttons is in `skins/laika/src/scss/overrides.scss` already where it is set to 270px. This scoped rule were not being used anyway because the file does not actually include a button, at least that's how I understand the `scoped` property. I double-checked and the buttons look the same before and after on all resolutions.